jedi_vim.py

```diff
@@ -453,16 +457,29 @@ def rename():
                 if vim.current.buffer.name != r.module_path:
                     result = new_buffer(r.module_path)
                     if not result:
-                        return
+                        echo_highlight("Jedi-vim: failed to create buffer window for {}!".format(r.module_path))
+                        continue
+
+                buffers.add(vim.current.buffer.name)
+
+                # Save view.
+                saved_view = vim_eval('winsaveview()')
 
                 vim.current.window.cursor = r.start_pos
                 vim_command('normal! cw%s' % replace)
```

Comment:
@davidhalter 
I think that should be changed to `c%dl%s` % (len(replace), replace)` (or similar).
Otherwise it might fail / do something unexpected, depending on the `iskeyword` setting.
Is there something better to be used from Jedi's results?
This should also handle multibyte strings properly, but I think it would do.